src/test/java/com/karl/http/HTTPStatusTest.java

```diff
@@ -0,0 +1,22 @@
+package com.karl.http;
+
+import org.junit.jupiter.api.Test;
+import static org.hamcrest.MatcherAssert.*;
+import static org.hamcrest.Matchers.*;
+
+public class HTTPStatusTest {
+  @Test
+  public void toStringReturnsTheCodeAndTheReasonPhraseSeperatedByAString() {
```

Comment:
What do you mean by "SeperatedByAString" at the end of the test name? Do you mean separated by a space?